test/Ssh.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+. $PSScriptRoot\Shared.ps1
+
+Describe 'SSH Function Tests' {
+    Context 'Get-SshPath Tests' {
+        It 'Returns the correct default path' {
+            Get-SshPath | Should BeExactly $Home\.ssh\id_rsa
+        }
+        It 'Returns the correct path given a filename' {
+            Get-SshPath mykey | Should BeExactly $Home\.ssh\mykey

```

Comment:
It shouldn't but I will test it.  PowerShell itself handles both `\` and `/` as path sep chars.  Issues arise though when a path is passed down to a native application.